### DISCUSSION TITLE
60 with: keyboard + predict; groups; morph inline

### DISCUSSION THREAD

**Discussion by sofia-bg** [UNC-Project-Open-AAC/OS-DPI#93] [Show and tell]
Feeling pretty good about this! 

Everything *seems* to be working, would help to have other eyes to see if anything is broken. 

~~[updated 60 3/14/22](https://github.com/UNC-Project-Open-AAC/OS-DPI/blob/9f55ac1ddc71e9a4b307435f3d6871c1d82bd17d/Designs%20In%20Progress/60_double_tap_morph_keyboard_groups_3_14_22.osdpi)~~

~~see [v2](https://github.com/UNC-Project-Open-AAC/OS-DPI/blob/3b67100ad24db1ede9b2a4ca451a17135f5af464/Designs%20In%20Progress/60_double_tap_morph_keyboard_groups_3_14_22_v2.osdpi) here~~

see [v3](https://github.com/UNC-Project-Open-AAC/OS-DPI/blob/f5f3b626b9927743567bc8a287288052435eb127/Designs%20In%20Progress/60_true_inline_3_14_22.osdpi) where now the groups, numbers wh- pages are all working -- and we have true "inline" for morph

@jbizovi @lageist @gbishop @frrancode

**Comment by lageist** [comment]
I'm getting the following error when I try to load file '60_double_tap_morph_keyboard_groups_3_14_22.osdpi'
![Screen Shot 2022-03-14 at 4 18 26 PM](https://user-images.githubusercontent.com/91340176/158254577-c752b1d1-0a50-4edf-a371-db7acbc211d5.png)

**Comment by jbizovi** [comment]
This has happened to me before but only when I try to load a file from a url.

Thank you,
Jenna

**Reply by lageist** [reply]
Did it load OK for you @jbizovi?

**Reply by sofia-bg** [reply]
Does this link work? Tried a fresh export. [vs2](https://github.com/UNC-Project-Open-AAC/OS-DPI/blob/4f8d314b4e9971c418a1831d032143a95e62a9d5/Designs%20In%20Progress/60_double_tap_morph_keyboard_groups_3_14_22_v2.osdpi)

**Reply by sofia-bg** [reply]
It was really laggy right now when I offloaded and imported the new one...

**Comment by jbizovi** [comment]
Nice work Sofia! I’m planning to look at this more tomorrow.

Thank you,
Jenna

**Reply by sofia-bg** [reply]
Make sure to use [v3](https://github.com/UNC-Project-Open-AAC/OS-DPI/blob/f5f3b626b9927743567bc8a287288052435eb127/Designs%20In%20Progress/60_true_inline_3_14_22.osdpi)

**Comment by sofia-bg** [comment]
Also as name implies - in order to see the morph you have to click the verb 2x. Going to spend time now doing a 2nd version where I actually indicate where the words should be so that each string would start where the original verb was.

**Reply by lageist** [reply]
new version you posted @sofia-bg loaded and working great to me now. Thanks.
